control-plane/pkg/reconciler/channel/channel_test.go

```diff
@@ -1349,6 +1350,71 @@ func TestReconcileKind(t *testing.T) {
 	useTable(t, table, env)
 }
 
+func TestFinalizeKind(t *testing.T) {
+
+	t.Setenv("SYSTEM_NAMESPACE", "knative-eventing")
+
+	messagingv1beta.RegisterAlternateKafkaChannelConditionSet(base.IngressConditionSet)
+
+	env := *DefaultEnv
+	testKey := fmt.Sprintf("%s/%s", ChannelNamespace, ChannelName)
+
+	table := TableTest{
+		{
+			Name: "Finalize normal - no auth",
```

Comment:
would you add more unit tests later or do you think only this is sufficient?

Reply:
It is sufficient for now I think.